core/src/com/po/fuck/Constants.java

```diff
@@ -18,4 +18,6 @@ public final class Constants {
 
     /* health bar */
     public static float HEALTHBAR_OFFSET = 5;
+    public static int CREEP_TEAM_TAG = 0;
```

Comment:
I think that it should be `ENEMY_TAG`, I'm not sure why is it a `CREEP`

Reply:
Also, I would make it final?